objects/src/transaction/tx_witness.rs

```diff
@@ -27,6 +32,7 @@ pub struct TransactionWitness {
     tx_inputs: TransactionInputs,
     tx_args: TransactionArgs,
     advice_witness: AdviceInputs,
+    signatures: BTreeMap<Digest, Vec<Felt>>,
```

Comment:
This should be its own newtype

Reply:
Could this not go into the `advice_witness`?